.github/workflows/ci.yml

```diff
@@ -24,5 +24,10 @@ jobs:
     - run: go test -mod=mod -coverpkg="./..." -coverprofile=covprofile ./...
     - env:
         COVERALLS_TOKEN: ${{ secrets.github_token }}
-      run: goveralls -coverprofile=covprofile -service=github
-
+      run: >-
+        goveralls -coverprofile=covprofile -service=github -ignore="
+        confgenerator/filter/internal/errors/*,
```

Comment:
It would make the package names even more verbose, but would it be better if we put all of these under `/filter/internal/generated/` so you would only have one directory to exclude?

Reply:
+1, I wasn't happy with this super long line, and that seems like a fine solution.